docs/usage/customization-guide.md

```diff
@@ -355,6 +355,34 @@ included in the list of accepted features.
 > tag is only evaluated in each re-discovery period, and the expiration of
 > node labels is not tracked.
 
+To exclude specific features from the `local.feature` Feature, you can use the
+`# +no-feature` directive. The `# +no-label` directive causes the feature to
+be excluded from the `local.label` Feature and a node label not to be generated.
+
+Considering the following file:
+
+```plaintext
+# +no-feature
+label-only=value
+
+my-feature=value
+
+foo=bar
+# +no-label
+foo=baz
+```
+
+Processing the above file would result in the following Features:
+
+```yaml
+local.features:
+  foo: baz
+  my-feature: value
+local.labels:
+  label-only: value
+  my-feature: value
+```
+
```

Comment:
Suggest an addition to mention the node labels, too (the indentation below is wrong to get the formatting right...)

```suggestion
    and the following labels added to the Node:
    
    ```
    feature.node.kubernetes.io/label-only=value
    feature.node.kubernetes.io/my-feature=value
    ```
```